commands/targets.go

```diff
@@ -0,0 +1,78 @@
+package commands
+
+import (
+	"log"
+	"os"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/concourse/fly/rc"
+	"github.com/concourse/fly/ui"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/fatih/color"
+)
+
+type TargetsCommand struct{}
+
+func (command *TargetsCommand) Execute([]string) error {
+	flyYAML, err := rc.LoadTargets()
+	if err != nil {
+		return err
+	}
+
+	if len(flyYAML.Targets) == 0 {
+		log.Fatalln("no targets found, please add some and try again")
```

Comment:
Don't bother checking for this. Having no targets saved is not an error; I'd expect to just see an empty table.